lib/rubocop/cop/bundler/ordered_gems.rb

```diff
@@ -17,7 +17,8 @@ module Bundler
       #
       #   gem 'rspec'
       class OrderedGems < Cop
-        MSG = 'Gem `%s` should appear before `%s` in their gem group.'.freeze
+        MSG = 'Gems should be sorted in an alphabetical order within their '\
```

Comment:
You've changed the string template, but you forgot to change the method using it. Moreover, I don't recall saying to drop the names - I think mentioning them would be useful.